src/components/CountDownButtonModal/index.tsx

```
@@ -20,30 +26,32 @@ export default function CountDownButtonModal({
   const [time, seconds] = textString.split(' ');
-  const { timeLeft, setTimeLeft, togglePause, isPaused } = useCountdown(time, true);
+  const { timeLeft, setTimeLeft, togglePause, isPaused } = useCountdown(parseInt(time), true);
 
-  useEffect(() => togglePause(), []);
+  useEffect(() => togglePause(), [togglePause]);
 
   const clickedButton = () => {
     preventParentClose();
     setOpen(true);
-    setTimeLeft(time);
+    setTimeLeft(parseInt(time));
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)